core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/MachineConfig.java

```diff
@@ -37,6 +37,20 @@
    */
   String MEMORY_REQUEST_ATTRIBUTE = "memoryRequestBytes";
 
+  /**
+   * Name of the attribute from {@link #getAttributes()} which, if present, defines the entrypoint
+   * command to be executed in the machine/container.
+   */
+  String CONTAINER_COMMAND_ATTRIBUTE = "containerCommand";
+
+  /**
+   * Name of the attribute from {@link #getAttributes()} which, if present, defines the command line
+   * arguments of the entrypoint command specified using the {@link #CONTAINER_COMMAND_ATTRIBUTE}.
+   *
+   * <p>If {@link #CONTAINER_COMMAND_ATTRIBUTE} is not present, this attribute is ignored.
```

Comment:
I'm not sure that it's good behavior. As far as I know, it's a normal scenario to specify only arguments for entry point from a container. It's like ENTRYPOINT and CMD in Dockerfile.
Please check if Kubernetes allows specifying arguments only. And let's do in the same way.

Reply:
:+1:  good point, will check